### ISSUE TITLE
Create keyboard shortcuts for Source kebab-menu items & display them in the menu

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-client#1358]
## Problem
Humans love keyboard shortcuts! Especially for frequently executed tasks. A few exist already, however In today's Client UI, nowhere do we show users what those are. While I would love to eventually create a slide-in panel to the far right of the Conversation Pane that would function as a cheat-sheet for all keyboard shortcuts (vs the menu bar path), the kebab (or "More") menu is a fine opportunity to display the shortcuts for its functions. 

## Proposed Solution
1. The scope of this issue is literally just for functionality that already exists at the time this issue is tackled, and to include the display of those functions in the menu. 
2. It's my opinion that it would be helpful to brainstorm what features we foresee being sought in the future; things such as Mute, Mark as Unread, Star/Unstar, etc., so that keyboard shortcuts might be devised that are as intuitive and memorable as possible. 
3. [Figma File](https://www.figma.com/file/YejZGCIGZikX55LDcfvcYY/Client-%E2%89%A5-Q4-2021?node-id=276%3A38469).

<img width="553" alt="Screen Shot 2021-11-27 at 1 44 31 PM" src="https://user-images.githubusercontent.com/8262612/143721790-a3500b24-d7e2-46df-a12f-893130a86674.png">

## Proposed Schema
_The below are my best guesses, partially informed by my own biases as a Mac user._

`Ctrl + P` = Print Source
`Ctrl + S` = Star Source (could eventually show in kebab menu to surface shortcut?)
`Ctrl + Alt + S` = Unstar Source
`Ctrl + Alt + E` = Export Source to USB
`Ctrl + E` = Export Source to Qube
`Ctrl + Delete` = Delete Source Messages/Files/Replies
`Ctrl + Alt + Delete` = Delete Entire Source Account (note: simply hitting "Delete" when a source is selected and in focus, should still delete a full Source; whereas if the cursor is in the Reply field, Ctrl-Alt-Del does the same thing)
`Ctrl + D` = Download All Source Files (not yet downloaded)
`Ctrl + M` = Mute Source
`Ctrl + U` = Mark Source Unread
`Ctrl + Return` = Send Reply
`Ctrl + T` = Create New Tag For Source

**Comment by ninavizz** [comment]
Related to #1354 & #1344

**Comment by eloquence** [comment]
See also #274 for tracking general keyboard navigation (though I think we can probably close that in favor of more specific issues like this one).

**Comment by ninavizz** [comment]
@eloquence I'd actually created #274 to facilitate the Tab, Tab, Tab, return, cursor keyboard-nav interaction, moreso than shortcuts—but if it'd be better to close that and then open specific issues for implementing A11y compliance, I'm cool with that!

**Comment by eloquence** [comment]
Yeah, what I mean is that it may make sense to close #274 in favor of specific issues like "Make tab ordering in source list more accessible" etc. But we can discuss that more yonder.

**Comment by gonzalo-bulnes** [comment]
> `Ctrl + S` = Star Source (could eventually show in kebab menu to surface shortcut?)
> `Ctrl + Alt + S` = Unstar Source

`Ctrl+S` is typically used for the "Save" action, which in SecureDrop context might be closer to "Download" or "Export" than it is to "Star"? Thinking aloud: "Star" seems closer to "Bookmark", doesn't it? (Typically `Ctrl+D`)

**Comment by gonzalo-bulnes** [comment]
**Reference documents on keyboard shortcuts / key bindings**

- https://developer.gnome.org/hig/reference/keyboard.html (list of shortcuts)
- https://website-archive.mozilla.org/www.mozilla.org/access/access/keyboard/ (guidelines)
- https://docs.microsoft.com/en-us/windows/win32/uxguide/inter-keyboard (access keys / shortcut keys)
